app/tests/integration/utils/filter_expression.js

```diff
@@ -0,0 +1,22 @@
+Components.utils.import("resource://gre/modules/components-utils/FilterExpressions.jsm");
+Components.utils.import("resource://gre/modules/components-utils/ClientEnvironment.jsm");
+Components.utils.import("resource://messaging-system/lib/RemoteSettingsExperimentLoader.jsm");
+
+// Targeting
+
+async function remoteSettings(arguments) {
+
+    const { TargetingContext } = ChromeUtils.import(
+        "resource://messaging-system/targeting/Targeting.jsm"
+    );
+    const ENVIRONMENT = {version: arguments[0], localeLanguageCode: "en-US"};
+    let targeting = new TargetingContext(ENVIRONMENT);
+    const EXPRESSION = `version|versionCompare('${arguments[1]}.!') >= 0 && localeLanguageCode == 'en'`;
```

Comment:
Should we add a couple tests for testing locales?

Reply:
I tried but locales don't work the way it would seem, at least for how we are doing it.

Reply:
Yeah I think the important part will be generating a DTO from the UI and testing the targeting expression it spits out against the browser.